examples/nidcpower_source_dc_voltage/_helpers.py

```diff
@@ -108,3 +111,57 @@ def get_grpc_device_channel(self, provided_interface: str) -> grpc.Channel:
                 service_class="ni.measurementlink.v1.grpcdeviceserver",
             ).insecure_address
         )
+
+
+class TestStandSupport(object):
+    """Class that communicates with TestStand."""
+
+    def __init__(self, sequence_context) -> None:
+        """Initialize the TestStandSupport object.
+
+        Args:
+            sequence_context:
+                The sequence context COM object from the TestStand sequence execution.
+
+        """
+        self._sequence_context = sequence_context
+
+    def set_active_pin_map_id(self, pin_map_id: str) -> None:
+        """Set the pin map ID to the MeasurementLink.PinmapId temporary variable.
+
+        Args:
+            pin_map_id (str):
+                The resource ID of the pin map that is registered to the pin map service.
+
+        """
+        self._sequence_context.Engine.TemporaryGlobals.SetValString(
+            "NI.MeasurementLink.PinMapId", 0x1, pin_map_id
+        )
+
+    def get_file_path(self, file_path: str) -> str:
+        """Return the full path of the input file if it is found in TestStand search directories.
+
+        Args:
+            file_path (str):
+                Name of the file to be found from the TestStand search directories.
+
+        Returns:
+            str: Specifies Absolute Path of the file.
+
+        """
+        sequence_file = self._sequence_context.SequenceFile
+        (_, abs_file_path, _, _, _) = self._sequence_context.Engine.FindFileEx(
+            file_path, searchContext=sequence_file
```

Comment:
This seems to return "." when you pass in an absolute file path or you click cancel on the file dialog, which is not what we want.

Reply:
Fixed:
- Check for absolute paths before calling FindFileEx
- Check userCancelled output